ocaml/idl/datamodel_host.ml

```diff
@@ -1668,6 +1668,35 @@ let cleanup_pool_secret =
       ]
     ~allowed_roles:_R_LOCAL_ROOT_ONLY ~hide_from_docs:true ()
 
+let host_numa_affinity_policy =
+  Enum
+    ( "host_numa_affinity_policy"
+    , [
+        ("any", "VMs are spread across all available NUMA nodes")
+      ; ( "best-effort"
```

Comment:
We don't have enough information: we attempt to set things up such that Xen choses NUMA placement the way we hope it would, but we cannot query what it actually did. Also if you try to start multiple VMs then due to race conditions one NUMA node may run out of memory and you'll get memory allocated from the other one anyway.

Also best effort because if we should never refuse to start a VM if it doesn't fit into its original NUMA nodes (we expand the set of NUMA nodes until it includes the entire host if needed)

Reply:
Updated the comment to say this is not guaranteed and may fall back to 'any' (we don't have any policies with guarantees atm)